src/useRaf.ts

```diff
@@ -22,7 +21,7 @@ const useRaf = (ms: number = 1e12, delay: number = 0): number => {
         cancelAnimationFrame(raf);
         set(1);
       }, ms);
-      start = Date.now();
+      start = performance.now();
```

Comment:
What do you think about keeping by default the original behavior, but allowing the user to overwrite it?

```suggestion
      start = now();
```

where

```ts
const useRaf = (ms: number = 1e12, delay: number = 0, now = Date.now): number => {
```

Reply:
To allow configuration of the starting time? Isn't that what the delay is for?